### ISSUE TITLE
Running `npm install realm@hermes` on Windows fails

### ISSUE THREAD

**Issue by hipaven** [realm/realm-js#4201]
### Description

npm install realm@hermes is getting error 

### Stacktrace & log output

```shell
npm install realm@hermes
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! code 1
npm ERR! path C:\source\repos\TestApp\node_modules\realm
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c prebuild-install -r napi || cmake-js build
npm ERR! prebuild-install WARN install No prebuilt binaries found (target=4 runtime=napi arch=x64 libc= platform=win32)
npm ERR! 'cmake-js' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
```

### Can you reproduce a bug?

Yes, always

### Reproduction Steps

npm install realm@hermes

### Version

Node - v16.13.1 React - 17.0.2 React -Native 0.66.4

### What SDK flavour are you using?

MongoDB Realm (i.e. Sync, auth, functions)

### Platform OS and version(s)

Windows  10.0.19043 Build 19043

**Comment by kneth** [comment]
Yes, we ran into an issue with our internal Windows builders (see the note in https://github.com/realm/realm-js/releases/tag/v10.20.0-beta.0).

**Comment by kraenhansen** [comment]
This is blocked on https://github.com/realm/realm-js/issues/4202

**Comment by kraenhansen** [comment]
We published the Windows binaries for the 10.20.0-beta.0 release and this should resolve this issue.
Please try again and reopen if the issue persists.
